src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -919,9 +919,9 @@ public DynamicJsonArray GetClusterErrors()
             return _engine.GetClusterErrorsFromLeader();
         }
 
-        public async Task<string> GenerateClusterIdentityAsync(string id, string databaseName)
+        public async Task<(string,long)> GenerateClusterIdentityAsync(string id, string databaseName)
```

Comment:
I think that other methods are returning Etag first, then the value.
Also use named tuple? (long Etag, string Id) ? Easier to work with.